test/draw/RasterCanvasTest.ooc

```diff
@@ -37,13 +37,13 @@ RasterCanvasTest: class extends Fixture {
 		})
 		this add("yuv420", func {
 			yuvImage := RasterYuv420Semiplanar open(this inputFlower)
-			correctImage := RasterRgb open("test/draw/correct/RasterCanvas_Yuv420.png")
 			for (i in 0 .. 30) {
-				pen := Pen new(ColorRgb new((i % 3) * 80, (i % 5) * 50, (i % 10) * 25))
+				pen := Pen new(ColorRgb new(255, 0, 0))
```

Comment:
A red color is easier to identify than random colors that were incorrect in the reference image.